src/screens/CreatePost/elements/SheetExpiredPost.js

```diff
@@ -4,53 +4,54 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
-import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
 
-const SheetExpiredPost = ({refExpired, data, select, onSelect}) => (
-    <RBSheet
-      ref={refExpired}
-      closeOnDragDown={true}
-      closeOnPressMask={true}
-      customStyles={{
-        container: styles.containerSheet,
-        draggableIcon: styles.draggableIcon,
-      }}>
-      <View style={styles.container}>
-        <Text style={styles.header}>Post expiration after</Text>
-        {data.map((value, index) => (
-          <FlatListItem
-            key={index}
-            value={value.label}
-            index={index}
-            select={select}
-            onSelect={onSelect}
-          />
-        ))}
-      </View>
-    </RBSheet>
-  );
+const SheetExpiredPost = ({refExpired, data, select, onSelect, isAnonimity = false}) => (
+  <RBSheet
+    ref={refExpired}
+    closeOnDragDown={true}
+    closeOnPressMask={true}
+    customStyles={{
+      container: styles.containerSheet,
+      draggableIcon: styles.draggableIcon
+    }}>
+    <View style={styles.container}>
+      <Text style={styles.header}>Post expiration after</Text>
+      {data.map((value, index) => (
+        <FlatListItem
+          key={index}
+          value={value.label}
+          index={index}
+          select={select}
+          onSelect={onSelect}
+          isAnonimity={isAnonimity}
+        />
+      ))}
+    </View>
+  </RBSheet>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `isAnonimity` has been added to the `SheetExpiredPost` component and passed down to the `FlatListItem` component. This change seems to be related to handling anonymous posts, but without further context or comments, it's hard to determine its exact purpose. Ensure that this new prop is properly documented and handled in the `FlatListItem` component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->